commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaService.java

```diff
@@ -13,13 +14,21 @@
 public class MfaService {
   private static final String DEFAULT_MFA_SYSTEM = "otp";
 
+  private static final String MFA_OIDC = "oidc";
+
+  private static final String MFA_FIDO = "fido";
```

Comment:
There properties should not be here. 
We have it in OTPConnector.type

The MFAService doesnt not know implementation details of connectors